### DISCUSSION TITLE
[BUG?] Searching private discussions using the GraphQL API returns no results when using app-issued token

### DISCUSSION THREAD

**Discussion by tefkah** [community/community#8853] [Discussions] [answered]
Just linking the discussion from Giscus, in case they haven't brought it up yet: https://github.com/giscus/giscus/issues/292#issuecomment-984362244

When making a search query on a private repository, no results are returned when using an app-issued token, whereas when using a private token via for instance the [GraphQL explorer](https://docs.github.com/en/graphql/overview/explorer) results are returned.

**Comment by trystanj** [comment]
Is this the right place to surface this bug / understand the difference in token scoping?

**Comment by LindseyB** [comment]
Thanks for the report. I've added this for our backlog to investigate and fix.

**Comment by antn** [comment, marked as answer]
Hi folks! I've deployed a fix for this, so Apps should be able to search for discussions in private repositories now, as long as the app has the `discussions: read` permission.

Please let us know if you still run into any issues!

**Reply by tefkah** [reply]
Amazing, thank you so much!
